docsrc/users-guide/examples/VI as Sampler Inits.ipynb

```diff
@@ -4,13 +4,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Using Variational Estimates to Initialize the NUTS-HMC Sampler\n",
+    "## Using Estimates from Variational, Laplace, or Optimization Methods to Initialize the NUTS-HMC Sampler\n",
     "\n",
-    "In this example we show how to use the parameter estimates return by Stan's variational inference algorithms\n",
-    "[pathfinder ](https://mc-stan.org/docs/cmdstan-guide/pathfinder-config.html)  and \n",
-    "[ADVI ](https://mc-stan.org/docs/cmdstan-guide/variational_config.html) \n",
-    "as the initial parameter values for Stan's NUTS-HMC sampler.\n",
-    "By default, the sampler algorithm randomly initializes all model parameters in the range uniform\\[-2, 2\\].  When the true parameter value is outside of this range, starting from the estimates from Pathfinder and ADVI will speed up and improve adaptation.\n",
+    "In this example, we show how to use parameter estimates returned by Stan's various posterior approximation or optimization algorithms as initial values for Stan's NUTS-HMC sampler. These include:\n",
```

Comment:
In this example we show how to use the returned estimates from a previous run of any of the inference algorithms can be used as initial values for  Stan's NUTS-HMC sampler.

Reply:
Updated.